developers/weaviate/config-refs/schema/index.md

```diff
@@ -150,6 +150,26 @@ To avoid this, you can either:
 
 We are working on a re-indexing API to allow you to re-index the data after adding a property. This will be available in a future release.
 
+### Collections count limit {#collections-count-limit}
+
+:::info
+Added in `v1.25.34`, `v1.26.18`, `1.27.15`, `1.28.9`, `1.29.2`
+:::
+
+To ensure optimal performance, Weaviate **limits the number of collections per instance**. Each collection adds overhead in terms of indexing, definition management, and storage. By setting a default limit, we help maintain performance and reduce operational complexity.
+
+- **Default limit**: `100` collections.
+- **Modify the limit**: Use the [`MAXIMUM_ALLOWED_COLLECTIONS_COUNT`](/developers/weaviate/config-refs/env-vars) environment variable to adjust the collection count limit.
+
+:::note 
+This change does not affect existing clusters that have already exceeded the default limit. However, it will prevent adding new collections if the limit has been reached.
```

Comment:
I don't think it is correct that this does not affect existing clusters exceeding the limit. (As collection creation is prevented.)

I suggest:

```info 
If the instance already exceeds the limit, Weaviate will not allow creation of any new collections. 
```